core/chains/evm/client/evm_client.go

```diff
@@ -37,3 +39,11 @@ func NewEvmClient(cfg evmconfig.NodePool, chainCfg commonclient.ChainConfig, cli
 	return NewChainClient(lggr, cfg.SelectionMode(), cfg.LeaseDuration(), chainCfg.NodeNoNewHeadsThreshold(),
 		primaries, sendonlys, chainID, chainType, clientErrors, cfg.DeathDeclarationDelay())
 }
+
+func getRPCTimeouts(chainType chaintype.ChainType) (largePayload, defaultTimeout time.Duration) {
+	if chaintype.ChainHedera == chainType {
```

Comment:
Instead of this strategy, isn't it more scalable, if we create 2 configs:
under ChainClient:
[ChainClient]
RPCTimeout = 10 seconds
LargeRPCTimeout = 10 seconds

Keep these default for all chains, except we set the LargeRPCTimeout = 30 for Hedera.

In future, updating these are just config changes not needing code fixes.